app/services/eventful_service.rb

```diff
@@ -0,0 +1,7 @@
+class EventfulService
+
+  def self.by_zipcode(token, zipcode)
```

Comment:
what purpose does passing in the token serve here if you've hardcoded the `app_key`?